sdk/core/azure-core-tracing-opencensus/src/samples/PublishEvents.md

```diff
@@ -14,7 +14,7 @@ Sample uses **[opencensus-impl][opencensus_impl]** as implementation package and
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-eventhubs</artifactId>
-  <version>5.0.0-beta.6</version>
+  <version>5.0.0-beta.7</version>
```

Comment:
I think this should be marked as a dependency version and we should update how these samples are injected into the README to ensure they compile and use the right version.